java/opendcs/src/main/java/ilex/gui/Help.java

```diff
@@ -2,25 +2,54 @@
 
 import javax.swing.*;
 import java.awt.Desktop;
+import java.io.File;
 import java.net.URI;
 import org.slf4j.LoggerFactory;
 
+
 public class Help
 {
     static final String HELP_URL = "https://opendcs-env.readthedocs.io/";
     private static final org.slf4j.Logger log = LoggerFactory.getLogger(Help.class);
+
+    private static String getUrl()
+    {
+        String localURL =System.getenv("DCSTOOL_HOME");
```

Comment:
Use getProperty. This is set automatically as a property now and may not exist and environment variable in a fresh installation.